### PR TITLE
NEXT-7087 #817 Fixed type error in SalesChannelCmsPageRepository sort…

### PR DESCRIPTION
**Pull request by BrentRobert**
### 1. Why is this change necessary?
The test only tested 1 section, therefore the sort was never executed and the type error was not clear. I fixed the test and afterwards the type error as well.

### 2. What does this change do, exactly?
It fixes a type issue.

### 4. Please link to the relevant issues (if any).
#817 

### 5. Checklist

- [x] I have written tests and verified that they fail without my change
- [x] I have squashed any insignificant commits
- [ ] I have written or adjusted the documentation according to my changes
- [ ] This change has comments for package types, values, functions, and non-obvious lines of code
- [x] I have read the contribution requirements and fulfil them.

### CODE REVIEW

**Comment by leichteckig** [comment]
QA approved :heavy_check_mark:

File: src/Core/Content/Test/Cms/SalesChannel/SalesChannelCmsPageRepositoryTest.php [hunk 1]
```diff
@@ -147,6 +147,21 @@ private function createPage(): string
                         ],
                     ],
                 ],
+                [
+                    'id' => Uuid::randomHex(),
+                    'type' => 'default',
+                    'position' => 1,
```
Comment (OliverSkroblin) [inline src/Core/Content/Test/Cms/SalesChannel/SalesChannelCmsPageRepositoryTest.php:154]: you trying to test the position sorting. Please set this position to 2 and add an assertion in the test for:

`$page->getSections()->first()->getPosition === 1`

**Comment by shyim** [comment]
Can you squash?

**Comment by BrentRobert** [comment]
Squashed! :D

**Review by shyim — APPROVED** [review]

**Comment by mitelg** [comment]
thanks for your contribution @BrentRobert :+1: :tada: :blue_heart:

**Pull request merged**